src/support/azext_support/_params.py

```diff
@@ -151,3 +151,20 @@ def load_communications_argument(self, _):
         c.argument('communication_body', help='Text of the communication.', required=True)
         c.argument('communication_subject', help='Subject of the communication.', required=True)
         c.argument('communication_sender', help='Email address of the sender.')
+
+def load_file_workspace_argument(self, _):
```

Comment:
This was needed for the old method, we dont need it anymore. Delete the changes here